linux/netfilter/netfilter.go

```diff
@@ -120,43 +121,56 @@ func (ctx* nftCtx) nftIfup(ifname string) {
     },
   })
 
-  // Inbound packets from outside the tunnel should be marked for RPF
+  // Inbound packets from wireguard servers should be marked for RPF
   // and moved into the external conntrack zone.
   ctx.conn.AddRule(&nftables.Rule{
     Table: ctx.table_inet,
     Chain: ctx.preroute,
     Exprs: []expr.Any{
-      // Match packets arriving from outside the tunnel
-			&expr.Meta{
-        Key:      expr.MetaKeyIIFNAME,
-        Register: 1,
+      // For now, we only support IPv4 in this check.
+      // TODO: Support IPv6.
```

Comment:
is this something we want to cover as follow up? Do we want to file a separate jira issue?

Reply:
We probably should, but the VPN client assumes IPv4 in many other areas of the code so it won't be a simple fix.